pycloudflare/models.py

```diff
@@ -112,19 +112,32 @@ def records(self):
             value.sort(key=lambda r: (r.type, r.content))
         return by_name
 
-    def create_record(self, name, type, content, ttl=1, proxied=False,
-                      priority=10):
+    def create_record(self, name, record_type, content=None, ttl=1,
+                      proxied=False, **kwargs):
         data = {
             'name': name,
-            'type': type,
-            'content': content,
+            'type': record_type,
             'ttl': ttl,
             'proxied': proxied,
         }
-        if type == 'MX':
-            data['priority'] = priority
 
-        record = self.service.create_dns_record(self.id, data)
+        if content:
+            data['content'] = content
+
+        if record_type == 'MX' and kwargs:
+            data['priority'] = kwargs['priority']
+        elif record_type == 'SRV' and kwargs:
+            data.update(
+                service=kwargs['service'],
+                name=name,
```

Comment:
@stefanor Right, that's a mistake. It'll be fine without this line. I'd forgotten about the initial data declaration.